build/circle-test.sh

```diff
@@ -69,8 +69,8 @@ prepare_artifacts() {
     head -n 100 "${outdir}/excerpt.txt"
   fi
 
-  if [ $ret -ne 0 ]; then
-    if [ "${CIRCLE_BRANCH-}" = "master" ] && [ -n "${GITHUB_API_TOKEN-}" ]; then
+  if [ $ret -ne 0 ] && [ -n "${GITHUB_API_TOKEN}" ]; then
```

Comment:
i dont understand this change; if GITHUB_API_TOKEN is unset, we won't report the correct exit code? seems bad.